plugins/defaultplugins/l2plugin/vppcalls/arp_term_vppcalls_test.go

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) 2017 Cisco and/or its affiliates.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package vppcalls
+
+import (
+	"testing"
+
+	govppapi "git.fd.io/govpp.git/api"
+	"github.com/ligato/cn-infra/logging/logrus"
+	l2ba "github.com/ligato/vpp-agent/plugins/defaultplugins/l2plugin/bin_api/l2"
+	"github.com/ligato/vpp-agent/plugins/defaultplugins/l2plugin/vppcalls/test/impl"
+	"github.com/onsi/gomega"
+)
+
+const (
+	dummyBridgeDomain uint32 = 4
+	dummyMACAddress          = "FF:FF:FF:FF:FF:FF"
+	dummyIPAddress           = "192.168.4.4"
+	dummyLoggerName          = "dummyLogger"
+)
+
+//TestVppAddArpTerminationTableEntry tests VppAddArpTerminationTableEntry
+func TestVppAddArpTerminationTableEntry(t *testing.T) {
+	gomega.RegisterTestingT(t)
+	mockedChannel := &impl.MockedChannel{Channel: govppapi.Channel{}}
+	VppAddArpTerminationTableEntry(dummyBridgeDomain, dummyMACAddress, dummyIPAddress, logrus.NewLogger(dummyLoggerName),
+		mockedChannel, nil)
+
+	var vppMsg l2ba.BdIPMacAddDel
+	switch t := mockedChannel.Msg.(type) {
```

Comment:
Using `switch` here is unnecessary. You can convert interface to desired type directly, like this:
```go
vppMsg, ok := mockedChannel.Msg.(*l2ba.BdIPMacAddDel)
if ok {
  // Msg is desired type..
} else {
  // Msg is not desired type..
}
```

Reply:
ok